docs/solidity-by-example.rst

```diff
@@ -1101,30 +1031,28 @@ The recipient should verify each message using the following process:
     2. Verify that the new total is the expected amount.
     3. Verify that the new total does not exceed the amount of Ether escrowed.
     4. Verify that the signature is valid and comes from the payment channel sender.
-    
+
 We'll use the `ethereumjs-util <https://github.com/ethereumjs/ethereumjs-util>`_
-library to write this verifications. The final step can be done a number of ways,
-but if it's being done in **JavaScript**.
-The following code borrows the `constructMessage` function from the signing **JavaScript code**
-above:
+library to write this verification. The final step can be done a number of ways,
+and we use JavaScript. The following code borrows the `constructMessage` function from the signing **JavaScript code** above:
 
 ::
 
     // this mimics the prefixing behavior of the eth_sign JSON-RPC method.
     function prefixed(hash) {
-        return ethereumjs.ABI.soliditySHA3(
+        return abi.soliditySHA3(
             ["string", "bytes32"],
             ["\x19Ethereum Signed Message:\n32", hash]
         );
     }
-    
+
     function recoverSigner(message, signature) {
-        var split = ethereumjs.Util.fromRpcSig(signature);
-        var publicKey = ethereumjs.Util.ecrecover(message, split.v, split.r, split.s);
-        var signer = ethereumjs.Util.pubToAddress(publicKey).toString("hex");
+        var split = ethereumjsutil.fromRpcSig(signature);
```

Comment:
Why change this? The initial version is how the browser builds of ethereumjs work.

Reply:
@axic Pretty much the same as the comment above.